taxonomies/ingredients.txt

```diff
@@ -22503,7 +22512,7 @@ vegetarian:en:yes
 en:durum wheat semolina
 de:Hartweizengrieß, Hartweizengriess, Hartweizengries
 el:σιμιγδάλισκληρού σιταριού
-es:sémola de trigo duro
+es:sémola de trigo duro, semola de trigo
```

Comment:
we already have a '"semola de trigo" entry:

<en:wheat
<en:semolina
en:wheat semolina
ar:ﺲﻤﻳﺩ
de:Weizengrieß, Weizengriess
es:sémola de trigo

Reply:
ok, missed the accent